paimon-core/src/main/java/org/apache/paimon/CoreOptions.java

```diff
@@ -1203,9 +1203,10 @@ public static StreamingReadMode fromValue(String value) {
 
     /** Compaction type when trigger a compaction action. */
     public enum StreamingCompactionType implements DescribedEnum {
-        NONE("none", "not a streaming compaction."),
-        NORMAL("normal", "Traditional bucket table compaction."),
-        NON_BUCKET("non-bucket", "Compaction for non-bucket table.");
```

Comment:
NONE("none", "not a streaming compaction."),
NORMAL("normal", "Traditional bucket table compaction."),
BUCKET_UNAWARE("non-bucket", "Compaction for non-bucket table.");